src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -198,6 +201,11 @@ private BoundExpression BindArgListOperator(InvocationExpressionSyntax node, Dia
                 else if (argument.Type.IsVoidType())
                 {
                     Error(diagnostics, ErrorCode.ERR_CantUseVoidInArglist, argument.Syntax);
+                    wasError = true;
+                }
+                else if (analyzedArguments.RefKind(i) == RefKind.None)
+                {
```

Comment:
why only RefKind.None, and not also RefKind.Ref?

Reply:
Binding to its natural type only makes sense for an rvalue.

---
In reply to: [301765261](https://github.com/dotnet/roslyn/pull/37052#discussion_r301765261) [](ancestors = 301765261)